hawkular-metrics/Dockerfile

```diff
@@ -75,15 +82,13 @@ COPY standalone.xml $JBOSS_HOME/standalone/configuration/standalone.xml
 # Overwrite the default logging.properties file
 COPY logging.properties $JBOSS_HOME/standalone/configuration/logging.properties
 
-# Install Hawkular-Metrics Python client, only available on EPEL
-USER root
-RUN yum -y install epel-release \
-    && yum -y install python-pip \
-    && yum clean all \
-    && pip install --no-cache-dir 'hawkular-client==0.4.4'
-
 # Change the permissions so that the user running the image can start up Hawkular Metrics
 RUN chmod -R 777 /opt
 USER 1000
 
-CMD $HAWKULAR_METRICS_SCRIPT_DIRECTORY/hawkular-metrics-wrapper.sh -b 0.0.0.0 -Dhawkular-metrics.cassandra-nodes=hawkular-cassandra
+## remove this ugly hack as soon as Hawkular ships with JGroups 3.6.12 or higher
+## we need it because of JGRP-2152
+RUN cd $JBOSS_HOME/modules/system/layers/base/org/jgroups/main && \
```

Comment:
This is critical: I'm not sure this works on EAP yet, but according to the mentioned JIRA, it would be available on the next EAP. Do we need to support a specific version of EAP? Can we require it to have "jgroups >= 3.6.12.Final" ?